config/locales/client.en.yml

```diff
@@ -5111,6 +5111,8 @@ en:
         install: "Install"
         delete: "Delete"
         delete_confirm: 'Are you sure you want to delete "%{theme_name}"?'
+        bulk_delete: 'Are you sure?'
+        bulk_delete_confirm: "This will uninstall the following themes / components, they will no longer be useable by any users on your site:"
```

Comment:
I think we can just pass a "type" to the modal to say "themes" or "components", instead of needing the slash, since we are only removing one or the other.